Assignments/SCF_Session-2/SCF_Assignment-1/src/HexCalculatorImplementation.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Scanner;
+
+public class HexCalculatorImplementation {
+	public static void main(String args[]){
+		Scanner sc=new Scanner(System.in);
+		String firstNumber="";
+		String secondNumber="";
+		String data="";
+		System.out.println("Enter you choice:");
+		System.out.println("1.Add  2.Subtract  3.Multiply  4.Divide  5.check equals  6.check greater than  7.check less than  8.convert into Decimal  9.conver into hex");
```

Comment:
@meta-vaibhav-gautam Line length should cross window. You have to break lines here.